mmv1/products/artifactregistry/Repository.yaml

```diff
@@ -116,3 +116,134 @@
         - :RELEASE
         - :SNAPSHOT
         default_value: :VERSION_POLICY_UNSPECIFIED
+  - !ruby/object:Api::Type::Enum
+    name: 'mode'
+    min_version: beta
+    description: |-
+      The mode configures the repository to serve artifacts from different sources.
+    input: true
+    values:
+      - :STANDARD_REPOSITORY
+      - :VIRTUAL_REPOSITORY
+      - :REMOTE_REPOSITORY
+    default_value: :STANDARD_REPOSITORY
+  - !ruby/object:Api::Type::NestedObject
+    name: 'virtualRepositoryConfig'
+    min_version: beta
+    conflicts:
+      - remote_repository_config
+    description: |-
+      Configuration specific for a Virtual Repository.
+    properties:
+      - !ruby/object:Api::Type::Array
+        name: 'upstreamPolicies'
```

Comment:
I do not find the field virtualRepositoryConfig  in the API doc https://cloud.google.com/artifact-registry/docs/reference/rest/v1beta2/projects.locations.repositories

Where is the API doc for the new fields? Thanks.

Reply:
v1 API docs were updated with this feature: https://cloud.google.com/artifact-registry/docs/reference/rest/v1/projects.locations.repositories

Reply:
Thanks.